### PR TITLE
NodeZooSDK (v1.0.1)

### PR DESCRIPTION
**Pull request by masture**
Please merge and review the NodeZooSDK.

### CODE REVIEW

File: Sources/NodeZooSDK/Internal/APIService.swift [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    func search(request: URLRequest, completion: @escaping (Result<[NZPackage], Error>) -> Void) -> URLSessionDataTask? {
+        
+        let dataTask = urlSession.dataTask(with: request) {data, response, error in
+            print("Received response")
+            guard let httpResponse = response as? HTTPURLResponse else {
+                completion(.failure(NodeZooServiceError.invalidResponse))
+                return
+            }
+            
+            // --- Debugging purpose only ---
```
Comment (rjrodger) [inline Sources/NodeZooSDK/Internal/APIService.swift:28]: Please add an option flag to turn this on and off

File: Sources/NodeZooSDK/Internal/NetworkManager.swift [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        print("Search request: \(searchRequest)")
+        
+        request.httpBody = searchRequest.body
+        
+        return apiService.search(request: request, completion: completion)
+        
+    }
+    
+    func createRequest() -> URLRequest? {
+        guard let url = URL(string: "https://api.nodezoo.com/api/public") else {
```
Comment (rjrodger) [inline Sources/NodeZooSDK/Internal/NetworkManager.swift]: also make this an option, with the current value as default

**Review by rjrodger — CHANGES_REQUESTED** [review]

File: Sources/NodeZooSDK/Internal/APIService.swift (hunk 1 above)
Comment (masture) [reply Sources/NodeZooSDK/Internal/APIService.swift:28]: Done

File: Sources/NodeZooSDK/Internal/NetworkManager.swift (hunk 2 above)
Comment (masture) [reply Sources/NodeZooSDK/Internal/NetworkManager.swift]: Done

**Pull request merged**